actnow/profiles/models.py

```diff
@@ -21,3 +21,20 @@ class UserProfile(models.Model):
 
     def __str__(self):
         return f"{self.first_name} {self.last_name}"
+
+
+class Organisation(models.Model):
+    name = models.CharField(max_length=20)
+    website_link = models.URLField(blank=True, null=True)
+    social_media_link = models.URLField(
+        validators=[validate_social_media_link], blank=True, null=True
+    )
+    person_1 = models.OneToOneField(
+        UserProfile, on_delete=models.SET_NULL, related_name="person_1", null=True
+    )
+    person_2 = models.OneToOneField(
+        UserProfile, on_delete=models.SET_NULL, related_name="person_2", null=True
+    )
```

Comment:
I'd rather we use `ManyToMany` here combined with something else ([`m2m_changed`](https://docs.djangoproject.com/en/3.2/ref/signals/#django.db.models.signals.m2m_changed) signal?) to limit the number to 2.